openerp/tools/amount_to_text_es.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+from translate import _
+
+_logger = logging.getLogger(__name__)
+
+# -------------------------------------------------------------
+# Español
+# -------------------------------------------------------------
+
+to_19 = ('Cero', 'Un', 'Dos', 'Tres', 'Cuatro', 'Cinco', 'Seis',
+         'Siete', 'Ocho', 'Nueve', 'Diez', 'Once', 'Doce', 'Trece',
+         'Catorce', 'Quince', 'Dieciséis', 'Diecisiete', 'Dieciocho',
+         'Diecinueve')
+tens = (
+'Veinte', 'Treinta', 'Cuarenta', 'Cincuenta', 'Sesenta', 'Setenta', 'Ochenta',
+'Noventa')
+denom = ('',
+         'Mil', 'Millón', 'Billón', 'Trillón', 'Cuatrillón',
+         'Quintillón', 'Sextillion', 'Septillion', 'Octillion', 'Nonillion',
+         'Decillion', 'Undecillion', 'Duodecillion', 'Tredecillion',
+         'Quattuordecillion',
+         'Sexdecillion', 'Septendecillion', 'Octodecillion', 'Novemdecillion',
+         'Vigintillion')
+
```

Comment:
Is it necessary to add the **_es** sufix to **to_19**, **tens** and **denom**? as it is done for the French and Dutch languages?